library/std/intrinsic.qs

```diff
@@ -712,7 +712,7 @@ namespace Microsoft.Quantum.Intrinsic {
     }
 
     /// # Summary
-    /// Applies the two qubit Ising $ZZ$ rotation gate.
+    /// Applies the two qubit Ising ZZ rotation gate.
```

Comment:
Some of these are quoted with ``, and some are not. I'm wondering which way is better. I think we shouldn't quote them as they aren't functions or keywords. This is just a normal text that used to be in "math style".
BTW, "math style" is typically italicized, maybe we can keep them in italics?

Reply:
I like the suggestion of italics.

Reply:
done